resources/validations/ui/src/context/cli/index.ts

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env -S ts-node --script-mode
+
+// @ts-ignore
+import * as SaxonJS from 'saxon-js';
+
+import { SaxonJsSchematronValidatorGateway } from '../shared/saxon-js-gateway';
+import { CommandLineController } from './cli-controller';
+import { readStringFile } from './file-gateway.humble';
+
+const controller = CommandLineController({
+  readStringFile,
+  validateSchematron: SaxonJsSchematronValidatorGateway({
+    sefUrl: `file:///Users/dan/src/10x/fedramp-automation/resources/validations/ui/public/ssp.sef.json`,
```

Comment:
B: I am not sure how to do this with Typescript when it is compiled down, but can we set up the equivalent of the `__dirname` functionality in Javascript.